models/mart/customer_status_daily_mrr.sql

```diff
@@ -0,0 +1,65 @@
+{{
+    config(
+        materialized='table'
+    )
+}}
+
+
+with customer_daily_mrr as (
+	SELECT 
```

Comment:
Other files use SQL keywords in downcase, its better to adopt a pattern

```suggestion
	select 
```